test/behavior.zig

```diff
@@ -143,6 +143,7 @@ test {
     _ = @import("behavior/bugs/13785.zig");
     _ = @import("behavior/bugs/14854.zig");
     _ = @import("behavior/bugs/15778.zig");
+    _ = @import("behavior/bugs/18302.zig");
```

Comment:
Do not add new `bugs/XXXX.zig` tests, instead place it one of the categorized files.

Reply:
Moved the test to `bit_shifting.zig`